src/components/Decks/components/DecksComponent.tsx

```diff
@@ -1,16 +1,39 @@
-import { Decks } from 'models/Deck'
+import { Deck, DeckName, Decks } from 'models/Deck'
 import './DecksComponent.scss'
+import { useState } from 'react'
 
 interface Props {
 	decks: Decks
 }
 
 const DecksComponent: React.FC<Props> = (props) => {
+	const [selected, setSelected] = useState<DeckName>('hiragana')
```

Comment:
Add `'hiragana'`, `'katakana'` and `'kanji'` into a constants inside a new `constants.tsx` file inside `utils`.

Then import them here, so if the value changes in a future we don't need to manually change them everywhere.

Reply:
done